src/variant_matrix/windows.cc

```diff
@@ -12,21 +12,24 @@ namespace Sequence
             }
```

Comment:
The window and slice functions now overlap significantly in their implementation.  A window is now just a slice involving all columns, and therefore `make_window` should simply call `make_slice.